Gemfile

```diff
@@ -13,7 +13,7 @@ gem 'jbuilder', '~> 2.7'
 gem 'bootsnap', '>= 1.4.2', require: false
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
-gem 'devise'
+gem 'devise' 
```

Comment:
This is an unnecessary change, right? Kindly remove this extra white space.

Reply:
resolved issue